### PR TITLE
feat(docs): Appointment letter

### PR DESCRIPTION
**Pull request by Anurag810**
screencapture-foo-bar-8000-docs-user-manual-en-human-resources-appointment-letter-2020-01-21-17_46_31

### CODE REVIEW

**Comment by Anurag810** [comment]
for feature https://github.com/frappe/erpnext/pull/20104

**Comment by nabinhait** [comment]
@prasadarr The dependent PR is merged, now you can review and merge this.

File: erpnext_com/www/docs/user/manual/en/human-resources/appointment-letter.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+1. Go to Appointment Letter Template list, click to New.
+1. fill Introduction, Terms and, Closing Statement.
+1. Click on Save.
+
+    <img class="screenshot" alt="Appointment Letter" src="{{docs_base_url}}/assets/img/human-resources/appointment-letter-template.png">
+
+> **Note** On selecting Appointment Letter Template in Appointment Letter it autofills the content.
+
+### 2.2 Print Format
+You can create or use existing standard print format.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/human-resources/appointment-letter.md]:
```suggestion
You can create new or use existing standard print formats.
```

File: erpnext_com/www/docs/user/manual/en/human-resources/appointment-letter.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+To create an Appointment Letter Template:
+
+1. Go to Appointment Letter Template list, click to New.
+1. fill Introduction, Terms and, Closing Statement.
+1. Click on Save.
+
+    <img class="screenshot" alt="Appointment Letter" src="{{docs_base_url}}/assets/img/human-resources/appointment-letter-template.png">
+
+> **Note** On selecting Appointment Letter Template in Appointment Letter it autofills the content.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/human-resources/appointment-letter.md]:
```suggestion
> **Note** On selecting Appointment Letter Template in Appointment Letter, it autofills the content.
```

File: erpnext_com/www/docs/user/manual/en/human-resources/appointment-letter.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+    <img class="screenshot" alt="Appointment Letter" src="{{docs_base_url}}/assets/img/human-resources/appointment-letter.png">
+
+> **Note:** Appointment Letter can be made only against a [Job Applicant](/docs/user/manual/en/human-resources/job-applicant)
+
+## 2. Features
+
+### 2.1 Appointment Letter Template
+
+To create an Appointment Letter Template:
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/human-resources/appointment-letter.md]:
```suggestion
```

File: erpnext_com/www/docs/user/manual/en/human-resources/appointment-letter.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+1. Go to Appointment Letter Template list, click to New.
+1. fill Introduction, Terms and, Closing Statement.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/human-resources/appointment-letter.md]:
```suggestion
1. Fill Introduction, Terms and, Closing Statement.
```

File: erpnext_com/www/docs/user/manual/en/human-resources/appointment-letter.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+1. Go to an Appointment Letter list, click on New.
+1. Select the Job Applicant and Appointment Date.
+1. You can manually fill Introduction, Terms and, Closing Statement or select an Appointment Letter Template to autofill the content.
+1. Click on Save.
+
+    <img class="screenshot" alt="Appointment Letter" src="{{docs_base_url}}/assets/img/human-resources/appointment-letter.png">
+
+> **Note:** Appointment Letter can be made only against a [Job Applicant](/docs/user/manual/en/human-resources/job-applicant)
+
+## 2. Features
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/human-resources/appointment-letter.md]: This should be 3. not 2.

File: erpnext_com/www/docs/user/manual/en/human-resources/appointment-letter.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+## 1. Prerequisites
+
+Before creating an Appointment Letter, it is advisable to create the following:
+
+* [Job Applicant](/docs/user/manual/en/human-resources/job-applicant)
+
+## 2. How to Create an Appointment Letter
+1. Go to an Appointment Letter list, click on New.
+1. Select the Job Applicant and Appointment Date.
+1. You can manually fill Introduction, Terms and, Closing Statement or select an Appointment Letter Template to autofill the content.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/human-resources/appointment-letter.md]:
```suggestion
1. You can manually fill Introduction, Terms and, Closing Statement or select an Appointment Letter Template to autofill the content. You can create new Appointment Letter Templates to select them easily.
```

File: erpnext_com/www/docs/user/manual/en/human-resources/appointment-letter.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+1. Click on Save.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/human-resources/appointment-letter.md]:
```suggestion
1. Click on Save.
1. After saving, go to the Print View to view and save the PDF of the Appointment letter.
```

File: erpnext_com/www/docs/user/manual/en/human-resources/appointment-letter.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
+<!-- add-breadcrumbs -->
+
+# Appointment Letter
+
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/human-resources/appointment-letter.md]:
```suggestion
> Introduced in Version 13
```

**Comment by Anurag810** [comment]
@prasadarr please merge

**Pull request merged**